R/helpers.R

```diff
@@ -454,3 +566,81 @@ checkDoubleArgumentsDesignObject <- function(designObj, ...) {
 
   }
 }
+
+# ---------- Boot helpers --------
+
+#' Computes the bootObj for sequential sampling procedures regarding nPlan, beta, the implied target
+#'
+#' @inheritParams designSafeZ
+#' @param values numeric vector. If objType equals "nPlan" or "beta" then values should be stopping times,
```

Comment:
Specify here that "Inf" should be passed instead of nPlan if H0 was not rejected after collecting nPlan data points in the simulation generating the values. Otherwise users of this function will not know ;)

Reply:
I changed the stoppingTimes to Inf before I pass them to this bootstrapping function. This is only done to compute beta, see for instance

https://github.com/AlexanderLyNL/safestats/blob/f35230d1646eee73958b22949dd5d496b445ff9f/R/zTest.R#L1570